api/db/migrations/20210715153308_add-column-schoolingRegistrationId-to-account-recovery-demands-table.js

```diff
@@ -0,0 +1,14 @@
+const TABLE_NAME = 'account-recovery-demands';
+const SCHOOLING_REGISTRATION_ID = 'schoolingRegistrationId';
+
+exports.up = function(knex) {
+  return knex.schema.table(TABLE_NAME, (table) => {
+    table.integer(SCHOOLING_REGISTRATION_ID).unsigned().references('schooling-registrations.id');
```

Comment:
Pour rester cohérent avec `TABLE_NAME`
```suggestion
const REFERENCED_COLUMN_NAME = ''schooling-registrations.id';
    table.integer(SCHOOLING_REGISTRATION_ID).unsigned().references(REFERENCED_COLUMN_NAME);
```

Reply:
done